packages/styled-components/components/pagination/index.js

```diff
@@ -0,0 +1,223 @@
+/* eslint max-len: 0 */
+import React from 'react';
+import PropTypes from 'prop-types';
+import queryString from 'query-string';
+import withThemes from '@irvingjs/styled/components/withThemes';
+import Link from 'components/link';
+import * as defaultStyles from './themes/default';
+
+/**
+ * Using the same props from a Pagination component, build the url for a given
+ * page.
+ *
+ * @param {object}  props Pagination component props.
+ * @param {integer} page  Page for the url.
+ * @return {string}
+ */
+const buildUrl = (props, page) => {
```

Comment:
This setup isn't perfect, but I'm comfortable with it as an alpha component.

Suggestions welcome, but they may get bumped to the next review stage.

Reply:
Agreed on both fronts. Adding a todo.